package.json

```diff
@@ -14,12 +14,16 @@
     "@grpc/proto-loader": "^0.7.13",
     "@react-navigation/bottom-tabs": "^6.5.20",
     "@react-navigation/native": "^6.1.17",
+    "@types/js-yaml": "^4.0.9",
+    "js-yaml": "^4.1.0",
     "long": "^5.2.3",
     "protobufjs": "^7.3.2",
     "react": "18.2.0",
     "react-native": "0.74.1",
+    "react-native-fs": "^2.20.0",
     "react-native-safe-area-context": "^4.10.4",
     "react-native-screens": "^3.31.1",
+    "yaml": "^2.4.5"
```

Comment:
missing comma here

Reply:
Always run the project then push the code as already told

Reply:
![Screenshot_20240629-154840.png](https://github.com/StarStrucks-Tech/Amaterasu/assets/108980694/6274bebb-a7f2-4389-8404-764aa82cf059)


I have run the project sir, that issue must happen by me while solving the conflicts.

Will be taken care from next time.